docker/ubuntu/koandroid/build_android_tc.sh

```diff
@@ -2,10 +2,10 @@
 
 echo "Pruning NDK to reduce image size..."
 NDK=android-ndk-r15c
-# only keep NDK 14
-rm -rf ${NDK}/platforms/android-{9,12,13,15,16,17,18,19,21,22,23,24,26}
-# only keep arm and x86
-rm -rf ${NDK}/toolchains/{aarch64-linux-android-4.9,mips64el-linux-android-4.9,mipsel-linux-android-4.9,x86_64-4.9}
+# only keep platform-14 (32bit ABIs) and platform-21 (64bit ABIs) 
+rm -rf ${NDK}/platforms/android-{9,12,13,15,16,17,18,19,22,23,24,26}
```

Comment:
Nothing higher than 26 in that newer one, like 27 or 28?

Reply:
Never mind, still the same NDK here. Minor mind hiccup wrt that issue from earlier about NDK 18.

Reply:
Yup. `platform` should match minSdkVersion for each ABI. Actually we're using `platform-14` everywhere, but https://github.com/koreader/android-luajit-launcher/pull/196 showcases how we need to use `platform-21` on 64bit abis